allprogram/6buttontable project.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+</head>
+<style>
+    button{
+        margin: 0.5cm;
+        padding: 10px;
+        font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+    }
+    
+</style>
+<body>
+   
+    <button name="1"  id="btn1"><b>1</b></button>
+    <button name="2" id="btn2"><p id="output2"></p><b>2</b></button>
+    <button name="3" id="btn3"><p id="output3"></p><b>3</b></button>
+    <button name="4" id="btn4"><p id="output4"></p><b>4</b></button>
+    <button name="5" id="btn5"><p id="output5"></p><b>5</b></button>
+    <button name="6" id="btn6"><p id="output6"></p><b>6</b></button>
+    <button name="7" id="btn7"><p id="output7"></p><b>7</b></button>
+    <button name="8" id="btn8"><p id="output8"></p><b>8</b></button>
+    <button name="9" id="btn9"><p id="output9"></p><b>9</b></button>
+    <button name="10" id="btn10"><p id="output10"></p><b>10</b></button>
+    <p  id="output1"></p>
+    
+    <script>
+        
+        $(document).ready(function(){
```

Comment:
function is not intended and remove unnecessary spacing . Add description of function.